client/src/routes/login.js

```diff
@@ -42,17 +48,37 @@ export default withStyles({
     }
   }
 
-  render (props, { teamToken, errors, disabledButton }) {
-    const { classes } = props
+  render ({ classes }, { teamToken, errors, disabledButton, ctftimeToken }) {
+    if (ctftimeToken) {
+      return <CtftimeAdditional ctftimeToken={ctftimeToken} />
```

Comment:
Ideally this would redirect to `/registration` while passing the token through using history state, however preact-router does not support attaching state (in JS) to a history entry. We can still implement this by implementing a state store using context and invalidate on any route change, which is enough to implement this.

Reply:
Documented in https://github.com/redpwn/rCTF/issues/201, out of scope for this PR though